ACP2E-346: [Documentation] Add final minimum(maximum) price calculation formula into number of documents (#1635)
magento/merchdocs: src/catalog/product-price-group.md

    ![]({% link images/images/storefront-cart-price-group-discount.png %}){: .zoom}
    _Group Price in Shopping Cart_

{:.bs-callout-tip}

> The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)

Reviewer: ```suggestion
The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

Author: Committed
